Code/Editor/MainWindow.cpp

```diff
@@ -845,78 +852,183 @@ void MainWindow::InitActions()
         .SetShortcut(tr("Z"))
         .SetToolTip(tr("Center on Selection (Z)"))
         .Connect(&QAction::triggered, this, &MainWindow::OnGotoSelected);
+
+    auto goToViewBookmark = [](int index)
+    {
+        if (AzToolsFramework::ViewBookmarkLoaderInterface* bookmarkLoader = AZ::Interface<ViewBookmarkLoaderInterface>::Get())
+        {
+            const AZStd::optional<AzToolsFramework::ViewBookmark> bookmark =
+                bookmarkLoader->GetBookmarkAtIndex(index, ViewBookmarkLoaderInterface::StorageMode::Local);
+
+            if (bookmark.has_value())
+            {
+                auto viewportContextManager = AZ::Interface<AZ::RPI::ViewportContextRequestsInterface>::Get();
+                if (!viewportContextManager)
+                {
+                    return false;
+                }
+
+                auto viewportContext = viewportContextManager->GetDefaultViewportContext();
+                if (!viewportContext)
+                {
+                    return false;
+                }
+
+                //Check the bookmark we want to load is not exactly 0;
+                if (bookmark.value() != ViewBookmark())
+                {
+                    SandboxEditor::InterpolateDefaultViewportCameraToTransform(
+                        bookmark->m_position, bookmark->m_rotation.GetX(), bookmark->m_rotation.GetZ());
+
+                    QString tagConsoleText = tr("View Bookmark %1 loaded position: x=%2, y=%3, z=%4")
+                                                 .arg(index + 1)
+                                                 .arg(bookmark->m_position.GetX(), 0, 'f', 2)
+                                                 .arg(bookmark->m_position.GetY(), 0, 'f', 2)
+                                                 .arg(bookmark->m_position.GetZ(), 0, 'f', 2);
+                    GetIEditor()->WriteToConsole(tagConsoleText.toUtf8().data());
+                    return true;
+                }
+            }
+        }
+
+        QString tagConsoleText = tr("Couldn't find View Bookmark Loader");
+        GetIEditor()->WriteToConsole(tagConsoleText.toUtf8().data());
+        AZ_Warning("Main Window", false, "goToViewBookmark: Couldn't find View Bookmark Loader");
+        return false;
+    };
+
     am->AddAction(ID_GOTO_LOC1, tr("Location 1"))
         .SetShortcut(tr("Shift+F1"))
-        .SetToolTip(tr("Location 1 (Shift+F1)"));
+        .SetToolTip(tr("Location 1 (Shift+F1)"))
+        .Connect(&QAction::triggered, [&goToViewBookmark](){ goToViewBookmark(0);});
```

Comment:
suggestion: I think these calls should be capturing `goToViewBookmark` by value as otherwise it will go out of scope and might cause issues, I think this is the safer thing to do for now (repeated below)